### ISSUE TITLE
Can't customize head.html after moving to SCSS

### ISSUE THREAD

**Issue by waterlens** [monkeyWzr/hugo-theme-cactus#82]
because the new code added to `head.html`:
```html
  {{- $options := (dict "targetPath" "css/styles.css" "outputStyle" "compressed" "enableSourceMap" "true") -}}
  {{- $styles := resources.Get "scss/style.scss" | resources.ExecuteAsTemplate "scss/style.scss" . | resources.ToCSS $options | resources.Fingerprint "sha512" }}
  <link rel="stylesheet" href="{{ $styles.Permalink }}" integrity="{{ $styles.Data.Integrity }}"> 
```
hugo reports errors when I copy `head.html` from the themes directory to the project directory

**Comment by Nizzlay** [comment]
@waterlens 

Can you please share the errors that you get?

**Comment by waterlens** [comment]
> @waterlens
> 
> Can you please share the errors that you get?

@Nizzlay 

Yeah. I guess that might be a problem related to relevant path?
```
ERROR 2021/07/08 XX:XX:XX Rebuild failed:
ERROR 2021/07/08 XX:XX:XX TOCSS: failed to transform "scss/style.scss" (text/x-scss): SCSS processing failed: file "stdin", line 2, col 1: File to import not found or unreadable: colors/<no value>.
```

**Comment by DaLeiGe** [comment]
Yes, I have the same bug
```txt
TOCSS: failed to transform "scss/style.scss" (text/x-scss): SCSS processing failed: file "stdin", line 2, col 1: File to import not found or unreadable: colors/<no value>.
```

**Comment by waterlens** [comment]
I found the problem.

```
  {{ .Scratch.Set "colortheme" "white"}}
  {{ if .Site.Params.Colortheme }}
    {{ .Scratch.Set "colortheme" .Site.Params.Colortheme }}
  {{ end }}
  {{ $colortheme := .Scratch.Get "colortheme" }}
```
In the past, this block of code will set the default  `$colortheme` if no `.Site.Params.Colortheme`, and we eventually use `$colortheme` to reference a css.
But now in `style.scss` `site.Params.Colortheme` was used directly, and there is no default value if we don't set it in `config.toml`.

**Comment by monkeyWzr** [comment]
@waterlens @daleige @Nizzlay Hi there
Really sorry for the inconvenience. This should be fixed by #88. Check it out!
Sorry again and best regards!

**Comment by waterlens** [comment]
Yeah, the problem has been solved.
